include/CLI/Formatter.hpp

```diff
@@ -6,7 +6,7 @@
 #include <string>
 
 #include "CLI/App.hpp"
-#include "CLI/Formatter.hpp"
+//#include "CLI/Formatter.hpp"
```

Comment:
The file seemed to be including itself not sure why?  Maybe should have been CLI/FormatterFwd.hpp?

Reply:
Yes, it was. Thanks!

Reply:
Just a note that the header was commented out in this PR,  you probably want to either remove it or change it to CLI/FormatterFwd.hpp in a subsequent revision.  It probably still works fine since the header is included in other locations before including this one.

Reply:
never mind I see that was done already